modules/graphql_core/src/Plugin/Deriver/Fields/ContextDeriver.php

```diff
@@ -45,6 +45,10 @@ public function getDerivativeDefinitions($basePluginDefinition) {
           'context_id' => $id,
           'type' => $context->getContextDefinition()->getDataType(),
         ] + $basePluginDefinition;
+        // Add cache contexts, if available
+        if (is_callable([$context, 'getCacheContexts'])) {
```

Comment:
Is there a better way to test the existence of this method? Check for an interface for instance?

Reply:
Hi @fubhy thanks for checking, yes I've added the check using `$context instanceof CacheableDependencyInterface` instead, hopefully that helps.